homeassistant/components/bluecurrent/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "domain": "bluecurrent",
+  "name": "BlueCurrent",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/bluecurrent",
+  "requirements": [],
```

Comment:
Zou de `bluecurrentapi` package niet nodig zijn als requirement of als dependency?